Makefile

```diff
@@ -34,19 +28,31 @@ help:
 .docker-build:
 	make build
 
+.devcontainer-build:
+	make devcontainerbuild
+
 .env:
 	./bin/cp-env-file.sh
 
 .PHONY: build
 build: .env  ## | Build docker images.
-	${DC} build --build-arg userid=${USE_UID} --build-arg groupid=${USE_GID} --progress plain base
+	${DC} build --progress plain base
```

Comment:
Oh I just noticed `make docs` still is looking for `USE_UID`, and is erroring locally. Can you remove that?

```diff
diff --git a/Makefile b/Makefile
index 937f475..d79b917 100644
--- a/Makefile
+++ b/Makefile
@@ -78,8 +78,8 @@ testshell: .env .docker-build  ## | Open shell in test environment.
 
 .PHONY: docs
 docs: .env .docker-build  ## | Build docs.
-	${DC} run --rm --user ${USE_UID} --no-deps eliot bash make -C docs/ clean
-	${DC} run --rm --user ${USE_UID} --no-deps eliot bash make -C docs/ html
+	${DC} run --rm --no-deps eliot bash make -C docs/ clean
+	${DC} run --rm --no-deps eliot bash make -C docs/ html
 
 .PHONY: lint
 lint: .env .docker-build  ## | Lint code.
```